css/webcharts.css

```diff
@@ -347,6 +347,8 @@
 		.wc-table table {
 			font-size: .9em;
 			border-collapse: collapse;
+			display: inline-table;
+			min-width: 550px;
```

Comment:
Yeah, since we removed the dynamic vertical vs. horizontal part js part, decided to have it always display horizontally and this is the minimum width (approximately) needed for the search bar and the sort text to show up together side by side. Jeremy and Spencer both suggested a minimum width on the table too #theteam #soughtcounsel. The vertical orientation was only needed when the width was less than that.

Reply:
So the search bar and sort text are a consistent width? That was my other concern with a magic number of pixels, beyond just the fact you hadn't used one anywhere before - if it was reliably going to be just enough, or you might be padding out something much smaller and having an odd visual effect.